lisabank-demo/src/test/java/com/ca/devtest/lisabank/demo/sv/http/BasicTest.java

```diff
@@ -85,10 +119,33 @@ public void getListUserTemplate() {
 		assertEquals(1, users.length);
 		printUsers(users);
 	}
+
+	@DevTestVirtualServiceV3(serviceName = "getListUserTemplateV3",
+			basePath = "/itkoExamples/EJB3UserControlBean",
+			port = "9081",
+			workingFolder = "UserServiceTest/getListUser/template",
+			inputFile2 = "listUser-1-req.xml",
+			inputFile1 = "listUser-1-rsp.xml",
+			parameters={@Parameter(name="email", value="pascal.gasp@gmail.com"),
+					@Parameter(name="nom", value="Gasp"),
+					@Parameter(name="login", value="pgasp"),
+					@Parameter(name="pwd", value="HELLO")},
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- fbfb66c4-c005-42c9-97fb-b90de4ccff9a -->